randovania/games/dread/exporter/patch_data_factory.py

```diff
@@ -351,6 +388,10 @@ def _tilegroup_patches(self):
         ]
 
     def create_data(self) -> dict:
+        # reset spawnpoint variables
+        self.spawnpoint_id = 0
```

Comment:
Oh, lol. Because what you wrote above: The variable is shared between all instances and exporting multiple seeds create(d) the wrong dict without resetting it. But this will be removed and is not needed when also moving `spawnpoint_id` and `new_spawn_points` into the `__init__` as you suggested

Reply:
Suspected such, but wasn't sure :)